spec/models/piece_spec.rb

```diff
@@ -225,44 +225,24 @@
       expect(king.y_position).to eq 3
     end
 
-    # it 'should move the piece to the destination square; if the square is occupied, mark that piece as captured' do
-    #   game = Game.create(white_user_id: 0, black_user_id: 1)
-    #   king = King.create(x_position: 4, y_position: 2, game_id: game.id)
-    #   rook = Rook.create(x_position: 4, y_position: 3, game_id: game.id)
-    #   king.move_to!(4,3)
-    #   expect(king.x_position).to eq 4
-    #   expect(king.y_position).to eq 3
-    #   expect(rook.captured).to eq true
-    # end
+  #   it 'should move the piece to the destination square; if the square is occupied, mark that piece as captured' do
+  #     white_user = FactoryGirl.create(:user)
+  #     black_user = FactoryGirl.create(:user)
+  #     game = Game.create(white_user_id: white_user.id, black_user_id: black_user.id)
+  #     king = King.create(x_position: 4, y_position: 2, game_id: game.id, user_id: white_user.id)
+  #     rook = Rook.create(x_position: 4, y_position: 3, game_id: game.id, user_id: black_user.id)
+  #     king.move_to!(4,3)
+  #     expect(king.x_position).to eq 4
+  #     expect(king.y_position).to eq 3
+  #     expect(rook.captured).to eq true
+  #   end
   end
 
-  describe 'white?' do
```

Comment:
@kyidkim  i know i wrote the black? and white? tests, i'm not sure about those other two.  i was just curious if you deleted them because they are failing now.  they seem unrelated to the current PR.